runtime/doc/pi_helpcurwin.txt

```diff
@@ -0,0 +1,65 @@
+*pi_helpcurwin.txt*	For Vim version 9.1.	Last change: 2025 Nov 30
+
+The helpcurwin |standard-plugin| enables opening help in the current window.
+
+1. :HelpCurwin			|helpcurwin-command|
+2. helpcurwin#Open()		|helpcurwin-function|
+3. <Plug>HelpCurwin;		|helpcurwin-mapping|
+
+This plugin is only available if 'compatible' is not set.
+You can avoid loading this plugin by setting the "g:loaded_helpcurwin"
+variable: >vim9
+
+	vim9script
+	g:loaded_helpcurwin = true
+<
+
+==============================================================================
+1. :HelpCurwin				     *:HelpCurwin* *helpcurwin-command*
+
+:HelpCurwin		Use the current window to display the help file,
+			|help.txt| in read-only mode.  It leaves any other
+			windows as-is (including when there is another
+			help window(s)).
+
+:HelpCurwin {subject}	Like ":HelpCurwin" but, additionally open the
+			applicable help file at the tag {subject}.
+			For example: >
+
+			  :HelpCurwin version9.2
+<
+			It should otherwise behave like :help {subject}.
+
+You may also want to save typing with a command line abbreviation,
+for example: >vim9
+
+	cnoreabbrev hc HelpCurwin
```

Comment:
Note, that this will expand `hc` in other contexts, this may not be desirable. Perhaps this is better:
```
cnoreabbrev <expr> hc getcmdtype() == ":" && getcmdline() == 'hc' ? 'HelpCurwin' : 'hc'

Reply:
Much better, yes.  It'll be there in the next commit.